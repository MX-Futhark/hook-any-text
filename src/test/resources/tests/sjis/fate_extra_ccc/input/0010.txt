82A6815B8141837B834E82CD23433233363139363030338357836983528181834A838A834D838A234344454681420A82B182CC8A77898082CC95E28C878BB388F582C581418DA182CD89C982F00A8E9D82C4975D82B582C482A282E991718CC98C7894F588F58148814082DD82BD82A282C881480002000000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F000000200000002100000022000000230000002400000025000000260000002700000028000000290000002A0000002B0000002C0000002D0000002E0000002F0000003000000B0000001415C90800000000000000000000000000008100000000000000000003001000FFFFFFFF0000000000000000FFFFFFDC000060C100001F43FFFFFFFF0000504300000000FFFFFFDC00000A4300001F43FFFFFFFF0000000000008843FFFFFFDC000060C100809B43FFFFFFFF0000504300008843FFFFFFDC00000A430B0000009415C90800000000000000000000000002008100000000000000000004001000FFFFFFFF0000000000000000FFFFFFDC000000C000002A43FFFFFFFF0000504300000000FFFFFFDC0000FC4200002A43FFFFFFFF0000000000008843FFFFFFDC000000C000009543FFFFFFFF0000504300008843FFFFFFDC0000FC4202000000000000000000000000000000000000001B00C201000000001013A3480B0000000000000000000000000000000000000002001000000000000000000000000000FFFFFFDC0000000000003C43000000000000F04300000000FFFFFFDC0000F04300003C4300000000000000000000A842FFFFFFDC0000000000008843000000000000F0430000A842FFFFFFDC0000F0430000884300000000000000000B00000000000000000000000000000000000000