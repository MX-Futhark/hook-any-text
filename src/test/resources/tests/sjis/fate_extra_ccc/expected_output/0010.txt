["えー、ボクはジナコ＝カリギリ。
この学園の補欠教員で、今は暇を
持て余している倉庫警備員？　みたいな？"]